core/builder/src/main/java/io/quarkus/builder/BuildContext.java

```diff
@@ -1,40 +1,29 @@
 package io.quarkus.builder;
 
-import static io.quarkus.builder.Execution.*;
-
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
-import java.util.Set;
 import java.util.concurrent.Executor;
-import java.util.concurrent.atomic.AtomicInteger;
 
 import org.wildfly.common.Assert;
```

Comment:
Can we finally use SmallRye's classes here?

Reply:
We could (and Qlue is) but I didn't want to change too much at once.  I could add a checklist item for it.